src/sdg/api/models.py

```diff
@@ -0,0 +1,30 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+
+class Token(models.Model):
+    """Custom authorization token model without binding to a specific user."""
+
+    key = models.CharField(_("key"), max_length=40, primary_key=True)
+    created = models.DateTimeField(_("created"), auto_now_add=True)
```

Comment:
We need some more information for a token. Can you add these fields as well: 

*  contact_person 
*    email 
*    organization 
*    application 
*    administration 
*    last_modified 

See: https://github.com/maykinmedia/objects-api/blob/master/src/objects/token/models.py#L14